plugins/talk-plugin-featured-comments/client/components/Tag.js

```diff
@@ -15,6 +16,20 @@ export default class Tag extends React.Component {
 
   }
 
+  componentDidMount() {
+    if (bowser.mobile) {
+      this.tagEl.addEventListener('touchstart', this.showTooltip);
```

Comment:
Why you don't use React events? https://facebook.github.io/react/docs/events.html#touch-events